security/crowdsec/src/opnsense/scripts/OPNsense/CrowdSec/reconfigure.py

```diff
@@ -27,6 +34,8 @@ def get_netloc(settings: dict[str, str]):
     # defaults if config has not been saved yet
     listen_address = settings.get('lapi_listen_address', '127.0.0.1')
     listen_port = settings.get('lapi_listen_port', '8080')
+    if is_ipv6(listen_address):
```

Comment:
I would urge to use a simple detection for ":" in the address for the same points as in core: the model (field type) ensures consistency